db/migrate/20231117134418_add_param_fields_to_planx_planning_data.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class AddParamFieldsToPlanxPlanningData < ActiveRecord::Migration[7.0]
+  def up
+    add_column :planx_planning_data, :params_v1, :jsonb
+    add_column :planx_planning_data, :params_v2, :jsonb
+
+    execute <<-SQL
+      UPDATE planx_planning_data
+      SET params_v1 = planning_applications.audit_log
```

Comment:
Have to make the code changes to use this new `params_v1` field on `planx_planning_data` and then eventually drop the audit_log on planning applications